tests/kafkatest/services/performance/end_to_end_latency.py

```diff
@@ -68,26 +70,12 @@ def start_cmd(self, node):
         })
 
         if node.version >= V_0_9_0_0:
-            """
-            val brokerList = args(0)
-            val topic = args(1)
-            val numMessages = args(2).toInt
-            val producerAcks = args(3)
-            val messageLen = args(4).toInt
-            """
-
             cmd = "KAFKA_OPTS=%(kafka_opts)s /opt/%(kafka_dir)s/bin/kafka-run-class.sh kafka.tools.EndToEndLatency " % args
-            cmd += "%(bootstrap_servers)s %(topic)s %(num_records)d %(acks)d %(message_bytes)d %(security_config_file)s" % args
+            cmd_args = "%(bootstrap_servers)s %(topic)s %(num_records)d %(acks)d %(message_bytes)d %(security_config_file)s"
+            if node.version >= TRUNK:
+                cmd_args += "%{compression_type}s"
+            cmd += cmd_args % args
         else:
-            """
-            val brokerList = args(0)
-            val zkConnect = args(1)
-            val topic = args(2)
-            val numMessages = args(3).toInt
-            val consumerFetchMaxWait = args(4).toInt
-            val producerAcks = args(5).toInt
-            """
```

Comment:
I removed this because I thought it didn't add much and people forget to keep it up to date when they modify the actual code. Let me know if you disagree.